SConstruct

```diff
@@ -284,102 +336,80 @@ if env['OS'] == 'Darwin':
+    defaults.apply("gcc")
     if env['OS'] == 'Cygwin':
         # See http://stackoverflow.com/questions/18784112
-        defaults.cxxFlags = '-std=gnu++11'
-    else:
-        defaults.cxxFlags = '-std=c++11'
-    defaults.buildPch = True
+        defaults.apply("Cygwin")
     env['pch_flags'] = ['-include', 'src/pch/system.h']
 
 elif env['CC'] == 'cl': # Visual Studio
-    defaults.cxxFlags = ['/EHsc']
-    defaults.ccFlags = ['/MD', '/nologo',
-                        '/D_SCL_SECURE_NO_WARNINGS', '/D_CRT_SECURE_NO_WARNINGS']
-    defaults.debugCcFlags = '/Zi /Fd${TARGET}.pdb'
-    defaults.noOptimizeCcFlags = '/Od /Ob0'
-    defaults.optimizeCcFlags = '/O2'
-    defaults.debugLinkFlags = '/DEBUG'
-    defaults.warningFlags = '/W3'
-    defaults.buildPch = True
+    defaults.apply("cl")
     env['pch_flags'] = ['/FIpch/system.h']
     env['openmp_flag'] = ['/openmp']
 
 elif 'icc' in env.subst('$CC'):
-    defaults.cxxFlags = '-std=c++11'
-    defaults.ccFlags = '-vec-report0 -diag-disable 1478'
-    defaults.warningFlags = '-Wcheck'
+    defaults.apply("icc")
     env['openmp_flag'] = ['-openmp']
 
 elif 'clang' in env.subst('$CC'):
-    defaults.ccFlags = '-fcolor-diagnostics'
-    defaults.cxxFlags = '-std=c++11'
-    defaults.buildPch = True
+    defaults.apply("clang")
     env['pch_flags'] = ['-include-pch', 'src/pch/system.h.gch']
 
 else:
     print("WARNING: Unrecognized C compiler '%s'" % env['CC'])
 
-if env['OS'] in ('Windows', 'Darwin'):
-    defaults.threadFlags = ''
-else:
-    defaults.threadFlags = '-pthread'
+if env['OS'] == 'Windows':
+    defaults.apply("Windows")
+elif env["OS"] == "Darwin":
+    defaults.apply("macOS")
 
 # InstallVersionedLib only fully functional in SCons >= 2.4.0
 # SHLIBVERSION fails with MinGW: http://scons.tigris.org/issues/show_bug.cgi?id=3035
 if (env['toolchain'] == 'mingw'
     or parse_version(SCons.__version__) < parse_version('2.4.0')):
-    defaults.versionedSharedLibrary = False
-else:
-    defaults.versionedSharedLibrary = True
-
-defaults.fsLayout = 'compact' if env['OS'] == 'Windows' else 'standard'
-defaults.env_vars = 'PATH,LD_LIBRARY_PATH,PYTHONPATH'
+    defaults.apply("mingw")
 
-defaults.python_prefix = '$prefix' if env['OS'] != 'Windows' else ''
-
-# Transform lists into strings to keep cantera.conf clean
-for key,value in defaults.__dict__.items():
-    if isinstance(value, (list, tuple)):
-        setattr(defaults, key, ' '.join(value))
+defaults.apply("default")
+defaults.python_cmd = sys.executable
 
 # **************************************
 # *** Read user-configurable options ***
 # **************************************
 
 config_options = [
     PathVariable(
-        'prefix',
-        'Set this to the directory where Cantera should be installed.',
+        "prefix",
+        """Set this to the directory where Cantera should be installed. On Windows
+           systems, '$ProgramFiles' typically refers to "C:\Program Files".""",
```

Comment:
Can you clarify what "typically" means here? If a user doesn't know what `'$ProgramFiles'` is or what it does, how should they interpret this help?

Reply:
This should become clear from the context once the ReST is assembled ... the corresponding portion is
```rest
.. _prefix-dev:

*  ``prefix``: [ ``path/to/prefix`` ]
   Set this to the directory where Cantera should be installed. On Windows
   systems, ``$ProgramFiles`` typically refers to ``"C:\Program Files"``.

   -  default: platform dependent

      -  Windows: ``'$ProgramFiles\Cantera'``
      -  Otherwise: ``'/usr/local'``
```

Reply:
OK, the ReST looks good, but does `scons help` return all that context too?

Reply:
Point taken!